mods/rv/rules/default-vehicles.yaml

```diff
@@ -4,11 +4,12 @@
 	Inherits@MC: ^MindControllableDisabledWithIC
 	Inherits@RADIATON: ^DamagedByRadiation10Percent
 	Inherits@SELECTION_MODE: ^LowPrioritySelectionMode
+	Inherits@AI: ^AIMicroManagable
```

Comment:
I try to keep these alphabetic (by the @ parts), can you move these at the beginning?